src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonerIndexService.kt

```diff
@@ -100,6 +112,25 @@ class PrisonerIndexService(
 
     prisonerDifferenceService.handleDifferences(existingPrisoner, offenderBooking, storedPrisoner)
 
+    log.trace("finished sync() {}", offenderBooking)
+    return storedPrisoner
+  }
+
+  fun reIndex(offenderBooking: OffenderBooking): Prisoner {
```

Comment:
this looks like nearly the same code as the `reIndex` - would it be worth moving into a function instead of duplicating?

Reply:
I had a go but its not really close enough to be able to improve much